src/v2/Apps/Fair/Components/ExhibitorsLetterNav.tsx

```diff
@@ -19,15 +18,7 @@ export const ExhibitorsLetterNav: React.FC<ExhibitorsLetterNavProps> = ({
   fair,
 }) => {
   const letters = fair?.exhibitorsGroupedByName?.map(group => group?.letter)
-  const {
-    height: [mobileNavBarHeight, desktopNavBarHeight],
-  } = useNavBarHeight()
-
-  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
-  const stickyTabsHeight = 150
-
-  const offset =
-    (isMobile ? mobileNavBarHeight : desktopNavBarHeight) + stickyTabsHeight
+  const offset = useExhibitorsTabOffset()
```

Comment:
Sorry if this is a dumb question, but I'm still working on understanding how SSR works in Force: is this component being rendered server-side? If so, I think we would want to avoid using this hook, right?

Reply:
I'm not sure, but I think you are right and we want to create a utility function instead of hook.
@dzucconi does it's right?

Reply:
Here it's not being used to _render_ anything so it's safe. That's sort of the distinction here: it's not SSR-safe but that's with regard to the actual rendering. Here it's being used to scroll to some position on screen after-mount; so all of that happens on the client-side.

Reply:
But yeah, I had mentioned we should just do this kind of thing _once_ everywhere since we're never not going to want to account for the sticky-stack when scrolling to a position. (Which reminds me to ticket it)

Reply:
thanks, that helps my understanding a lot!